src/componants/Article.jsx

```diff
@@ -27,8 +28,14 @@ function Article (){
       setCategory(results.topic)
       setLoaded(true)
     })
+    .catch((error) => {
+      setError(true)
+    })
   }, [])
 
+  if(error){
+    return (<h2>This article does not exist</h2>)
+    }
```

Comment:
You could also use the error message that comes back from your API if you save the error object in state